magpie-run

```diff
@@ -1042,6 +1042,165 @@ else
+	sleep ${sleepwait}
+	totalsleepwait=`expr ${totalsleepwait} + ${sleepwait}`
+    fi
+
+    echo "*******************************************************"
+    echo "*"
+    echo "* Phoenix Information"
+    echo "*"
+    echo "* To access phoenix directly, you'll want to:"
+    echo "*   ${MAGPIE_REMOTE_CMD:-ssh}${MAGPIE_REMOTE_CMD_OPTS:+" "}${MAGPIE_REMOTE_CMD_OPTS} ${HBASE_MASTER_NODE}"
+    if echo $MAGPIE_SHELL | grep -q csh
+    then
+	echo "*   setenv HBASE_CONF_DIR \"${HBASE_CONF_DIR}\""
+    else
+	echo "*   export HBASE_CONF_DIR=\"${HBASE_CONF_DIR}\""
+    fi
+    echo "*   cd $PHOENIX_HOME"
+    echo "*"
+    echo "* Then you can do as you please.  To interact with Phoenix you can use:"
+    echo "*" 
+    echo "*   ${phoenixcmdprefix}/sqlline.py"
+
+    echo "*" 
+    echo "* Some additional environment variables you may sometimes wish to set"
+    echo "*" 
+    if echo $MAGPIE_SHELL | grep -q csh
+    then
+	echo "*   setenv JAVA_HOME \"${JAVA_HOME}\""
+	echo "*   setenv HBASE_HOME \"${HBASE_HOME}\""
+        echo "*   setenv PHOENIX_HOME=\"${PHOENIX_HOME}\""
+    else
+	echo "*   export JAVA_HOME=\"${JAVA_HOME}\""
+	echo "*   export HBASE_HOME=\"${HBASE_HOME}\""
+        echo "*   export PHOENIX_HOME=\"${PHOENIX_HOME}\""
+    fi
+
+    if [ "${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}X" != "X" ]
+    then
+	echo "*"
+	echo "* If running interactively, sourcing"
+	echo "*"
+	echo "* ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}"
+	echo "*"
+	echo "* will set most common environment variables for your job."
+    fi
+    echo "*" 
+    echo "*******************************************************"
+
+    if [ "${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}X" != "X" ]
+    then
+	if echo $MAGPIE_SHELL | grep -q csh
+	then
+	    echo "setenv HBASE_HOME \"${HBASE_HOME}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "setenv HBASE_CONF_DIR \"${PHOENIX_CONF_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "setenv HBASE_LOG_DIR \"${PHOENIX_LOG_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "setenv HBASE_MASTER_NODE \"${HBASE_MASTER_NODE}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "setenv HBASE_REGIONSERVER_COUNT \"${HBASE_REGIONSERVER_COUNT}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+            echo "setenv PHOENIX_HOME=\"${PHOENIX_HOME}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	else
+	    echo "export HBASE_HOME=\"${HBASE_HOME}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "export HBASE_CONF_DIR=\"${PHOENIX_CONF_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "export HBASE_LOG_DIR=\"${PHOENIX_LOG_DIR}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "export HBASE_MASTER_NODE=\"${HBASE_MASTER_NODE}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	    echo "export HBASE_REGIONSERVER_COUNT=\"${HBASE_REGIONSERVER_COUNT}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+            echo "export PHOENIX_HOME=\"${PHOENIX_HOME}\"" >> ${MAGPIE_ENVIRONMENT_VARIABLE_SCRIPT}
+	fi
```

Comment:
Yes, I believe you are correct for the most part, however Phoenix relies on some HBase vars to be set. Since I attempted to keep HBase and Phoenix separate I had to change a few of the HBase vars like the conf and log dir. They point to a modified version of HBases original setup.